types/base/api.d.ts

```diff
@@ -0,0 +1,42 @@
+type Part = {
+    name: string;
+    material: string;
+    color: string;
+}
+
+type Spec = {
+    brand: string;
+    model: string;
+    parts: Array<Part>;
+    initials: string;
+    engraving: string;
+    initialsExtra: object;
+    variant?: string;
+    version?: string;
+    description?: string;
+};
+
+type ProductionOrderOptions = {
+    brand?: string;
+    model?: string;
+    variant?: string;
+    parts?: Array<Part>;
+    initials?: string;
+    engraving?: string;
+    initialsExtra?: object;
+    gender?: string;
+    size?: number;
+    pending?: boolean;
+    notify?: boolean;
+    productId?: string;
```

Comment:
number? not sure

Reply:
Im confused, i see some string instances:
- https://github.com/ripe-tech/plugin-farfetch/blob/2cb3979de84ad7cbd7f5b00875384a141271c837/static/themes/farfetch/js/steps/mkfbpixel.part.js#L54
- https://github.com/ripe-tech/plugin-farfetch/blob/2cb3979de84ad7cbd7f5b00875384a141271c837/static/themes/farfetch/js/steps/mkfbpixel.part.js#L54

But I see in Core this cast to int:
- https://github.com/ripe-tech/plugin-farfetch/blob/2cb3979de84ad7cbd7f5b00875384a141271c837/static/themes/farfetch/js/steps/mkfbpixel.part.js#L54

Reply:
String seems the "better safe than sorry" approach

Reply:
It's a `number`: https://github.com/ripe-tech/ripe-core/search?q=product_id